docs/src/content/docs/reference/frontmatter.md

```diff
@@ -29,6 +29,12 @@ You must provide a title for every page. This will be displayed at the top of th
 
 The page description is used for page metadata and will be picked up by search engines and in social media previews.
 
+### `slug`
+
+**type**: `string`
+
+Content collections reserved property to override the slug of the page, check [defining custom slugs](https://docs.astro.build/en/guides/content-collections/#defining-custom-slugs) for more details.
```

Comment:
Thanks for the suggestion @chrskerr! In general we try not to document too much that’s part of Astro’s default functionality, but I agree in this case it’s helpful. Going to make one slight tweak to the proposed text and then should be good to merge.

```suggestion
Override the slug of the page, see [“Defining custom slugs”](https://docs.astro.build/en/guides/content-collections/#defining-custom-slugs) in the Astro docs for more details.
```